Common/MathUtils/include/MathUtils/CartesianGPU.h

```diff
@@ -26,8 +26,8 @@ template <typename T, int I>
 struct GPUPoint2D {
   GPUdDefault() GPUPoint2D() = default;
   GPUd() GPUPoint2D(T a, T b) : xx(a), yy(b) {}
-  GPUd() float X() const { return xx; }
-  GPUd() float Y() const { return yy; }
+  GPUhd() float X() const { return xx; }
```

Comment:
@davidrohr please see these kind of changes. Here I think they are harmless.

Reply:
In principle they should be harmless for inline functions, not to be used for non-inline functions.
But basically, the host version is only needed if it is called by the host part during GPU compilation, which should usually not be the case?